grid2op/gym_compat/discrete_gym_actspace.py

```diff
@@ -252,6 +252,7 @@ def __init__(
             "curtail": act_sp.get_all_unitary_curtail,
             "curtail_mw": act_sp.get_all_unitary_curtail,
             "raise_alarm": act_sp.get_all_unitary_alarm,
+            "raise_alert": act_sp.get_all_unitary_alert,
```

Comment:
I don't think it makes a lot of sense. Indeed, the alert is, by essence, "per line". It does not really make much sens, in my opinion, to encode it in gym like a "unique integer". It make more sense to have a multibinary.

Reply:
So i'm not sure it supposed to be in the "discrete_gym_actspace" at all.

Reply:
At the very least there should be a warning "you should really not do that" or something.